diff -x '*~' -u -r a/chrome/content/extensions.rdf b/chrome/content/extensions.rdf
--- a/chrome/content/extensions.rdf	2013-09-26 15:13:11.000000000 +0200
+++ b/chrome/content/extensions.rdf	2013-12-20 19:08:00.342190978 +0100
@@ -5,7 +5,7 @@
   xmlns:em="http://www.mozilla.org/2004/em-rdf#"
   xmlns:NC="http://home.netscape.com/NC-rdf#">
   <Seq about="http://inverse.ca/sogo-integrator/extensions"
-    isi:updateURL="http://sogo-demo.inverse.ca/plugins/updates.php?plugin=%ITEM_ID%&amp;version=%ITEM_VERSION%&amp;platform=%PLATFORM%">
+    isi:updateURL="{ $updateURL }">
     <li>
       <Description
         em:id="sogo-integrator@inverse.ca"
diff -x '*~' -u -r a/defaults/preferences/site.js b/defaults/preferences/site.js
--- a/defaults/preferences/site.js	2013-09-26 15:13:11.000000000 +0200
+++ b/defaults/preferences/site.js	2013-12-20 19:12:14.341344085 +0100
@@ -9,5 +9,5 @@
 pref("mail.strictly_mime.parm_folding", 0);
 pref("calendar.alarms.showmissed", false);
 pref("calendar.caldav.sched.enabled", true);
-pref("calendar.timezone.local", "/mozilla.org/20070129_1/America/Montreal");
-pref("sogo-integrator.autocomplete.server.urlid", "public");
+pref("sogo-integrator.autocomplete.server.urlid", "users");
