diff -x '*~' -u -r a/chrome/content/extensions.rdf b/chrome/content/extensions.rdf
--- a/chrome/content/extensions.rdf	2012-08-24 22:39:09.000000000 +0200
+++ b/chrome/content/extensions.rdf	2013-02-06 15:35:46.976630345 +0100
@@ -5,7 +5,7 @@
   xmlns:em="http://www.mozilla.org/2004/em-rdf#"
   xmlns:NC="http://home.netscape.com/NC-rdf#">
   <Seq about="http://inverse.ca/sogo-integrator/extensions"
-    isi:updateURL="http://sogo-demo.inverse.ca/plugins/updates.php?plugin=%ITEM_ID%&amp;version=%ITEM_VERSION%&amp;platform=%PLATFORM%">
+    isi:updateURL="{ $updateURL }">
 <!--  <li>
       <Description
         em:id="lookout@aron.rubin"
diff -x '*~' -u -r a/defaults/preferences/site.js b/defaults/preferences/site.js
--- a/defaults/preferences/site.js	2012-08-24 22:37:33.000000000 +0200
+++ b/defaults/preferences/site.js	2013-02-06 15:36:42.358363876 +0100
@@ -8,7 +8,7 @@
 pref("mail.mdn.report.not_in_to_cc", 0);
 pref("mail.strictly_mime.parm_folding", 0);
 pref("sogo-integrator.disable-calendaring", false);
-pref("sogo-integrator.autocomplete.server.urlid", "public");
+pref("sogo-integrator.autocomplete.server.urlid", "users");
 pref("javascript.options.showInConsole", true);
 pref("nglayout.debug.disable_xul_cache", true);
 pref("browser.dom.window.dump.enabled", true);
